ui/app/helpers/utils/i18n-helper.js

```diff
@@ -39,13 +40,22 @@ export const getMessage = (localeCode, localeMessages, key, substitutions) => {
   }
   const entry = localeMessages[key]
   let phrase = entry.message
+
+  const hasSubstitutions = Boolean(substitutions && substitutions.length)
+  const hasReactSubstitutions = hasSubstitutions &&
+    substitutions.some((element) => typeof element === 'function' || typeof element === 'object')
+
   // perform substitutions
-  if (substitutions && substitutions.length) {
-    substitutions.forEach((substitution, index) => {
-      const regex = new RegExp(`\\$${index + 1}`, 'g')
-      phrase = phrase.replace(regex, substitution)
-    })
+  if (hasSubstitutions) {
+    const parts = phrase.split(/(\$\d)/g)
+
+    const substitutedParts = parts.map((part) => (part.match(/\$\d/) ? substitutions.shift() : part))
```

Comment:
This would no longer allow using the same substitution twice in the same message. We need to be matching the specific indexes - e.g. `$1` should always refer to the first in the `substitutions` list, even if it isn't first in the localized message itself.

Reply:
good catch

Fixed in 5a679aa33